### PR TITLE
Fixing Highlighted modifier to card

### PR DESCRIPTION
**Pull request by eduardojmatos**
**CHANGELOG** :memo:

* We don't need italic style to `card__content`. Removing this property and applying to title classes.

### CODE REVIEW

File: docs/demo/gaiden-css/components/card/centered.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+  <style>
+    body {
+      background-color: #eff4fa;
```
Comment (thiagodelmotte) [inline docs/demo/gaiden-css/components/card/centered.html]: @eduardojmatos dúvida: porque essa cor não pode ficar nos stylesheets?

File: src/scss/components/_card.scss [hunk 2]
```diff
@@ -45,7 +59,8 @@
   }
 
   &--highlighted {
-    .card__content {
+    [class*='-title'],
+    [class*='__title'] {
       color: get-color(rastaGray);
```
Comment (thiagodelmotte) [inline src/scss/components/_card.scss]: @eduardojmatos rastaGray?

File: docs/demo/gaiden-css/components/card/centered.html (hunk 1 above)
Comment (eduardojmatos) [inline docs/demo/gaiden-css/components/card/centered.html]: @thiagodelmotte this is only a docs file. The point here is to evidence the cards shadow effect on the page.

File: src/scss/components/_card.scss (hunk 2 above)
Comment (eduardojmatos) [inline src/scss/components/_card.scss]: @thiagodelmotte yep... this names was defined by design team

Comment (bsoares) [inline src/scss/components/_card.scss]: Deep elements with a class that contains `title` in the name, will be affected with this css.

**Pull request merged**